metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/dao/DepublishRecordIdDao.java

```diff
@@ -133,7 +133,7 @@ public int createRecordIdsToBeDepublished(String datasetId, Set<String> candidat
     return recordIdsToAdd.size();
   }
```

Comment:
In this class, there are several interesting lines of code that don't have test coverage. Perhaps you could look into some of those?

Reply:
Created new unit tests for the `if else` clauses